products/workflows/frontend/Workflows/hogflows/panel/testing/hogFlowEditorNotificationTestLogic.ts

```
@@ -0,0 +1,367 @@
+                loadSamplePersonByDistinctIdSuccess: () => false,
+                loadSamplePersonByDistinctIdFailure: () => false,
+            },
+        ],
+        sampleGlobalsError: [
+            null as string | null,
+            {
+                setSampleGlobalsError: (_, { error }) => error,
+                loadSamplePersonByDistinctIdSuccess: () => null,
+            },
+        ],
+    }),
+    loaders(() => ({
+        samplePersons: [
+            [] as PersonType[],
+            {
+                loadSamplePersons: async () => {
+                    const response = await api.persons.list({ limit: 5 })
+                    if (!response.results || response.results.length === 0) {
+                        return []
+                    }
+                    return response.results
+                },
+            },
+        ],
+        personSearchResults: [
+            [] as PersonType[],
+            {
+                searchPersons: async ({ searchTerm }) => {
+                    if (!searchTerm.trim()) {
+                        return []
+                    }
+                    try {
+                        const response = await api.persons.list({ search: searchTerm.trim(), limit: 10 })
+                        return response.results
+                    } catch (error) {
+                        console.error('Failed to search persons:', error)
+                        return []
+                    }
+                },
+            },
+        ],
+    })),
+    forms(({ actions, values }) => ({
+        testInvocation: {
+            defaults: {
+                mock_async_functions: false,
+            } as HogflowTestInvocation,
+            errors: (data: HogflowTestInvocation) => {
+                const errors: Record<string, string> = {}
+                try {
+                    JSON.parse(data.globals)
+                } catch {
+                    errors.globals = 'Invalid JSON'
```

same here i think  `posthog.captureException` could be of use here otherwise we also swallow and mask the error